support/src/main/java/se/inera/intyg/common/support/facade/util/ValueToolkit.java

```diff
@@ -77,6 +77,10 @@ public static String textValue(Map<String, CertificateDataElement> data, String
             return null;
         }
 
+        if (textDataValue.getText() == null || textDataValue.getText().isEmpty()) {
```

Comment:
Good if we have tests on this behaviour in the ValueToolkitTest-class also.

Reply:
Ops... fixed it now! Added tests that verify null & empty string behaviour.